assets/js/modules/analytics-4/components/widgets/TopCategoriesWidget.stories.js

```diff
@@ -133,6 +301,28 @@ export default {
 					},
 				] );
 
+				provideModuleRegistrations( registry );
+
+				const [ accountID, propertyID, webDataStreamID ] = [
+					'12345',
+					'34567',
+					'56789',
+				];
```

Comment:
What's the benefit here? Would it not be less lines to do a normal assignment for each?